### PR TITLE
Bifrost: windows auto-updater

### PR DESCRIPTION
**Pull request by darakeon**
Auto-updater for windows

Little fixes of details noticed in first user test

### CODE REVIEW

File: PagarMe.Bifrost.Certificates/PagarMe.Bifrost.Certificates.csproj [hunk 1]
```diff
@@ -49,6 +49,9 @@
     <Reference Include="System.Xml" />
```
Comment (wilkmaia) [inline PagarMe.Bifrost.Certificates/PagarMe.Bifrost.Certificates.csproj]: Squash with the other Assembly commit.

File: PagarMe.Bifrost.Certificates/Generation/CertificateGenerator.cs [hunk 2]
```diff
@@ -112,7 +112,7 @@ private void setPrivateKey(X509Certificate2 x509, AsymmetricCipherKeyPair keyPai
             };
```
Comment (wilkmaia) [inline PagarMe.Bifrost.Certificates/Generation/CertificateGenerator.cs]: Replace `to` with `so` on commit message.

File: PagarMe.Bifrost/Updates/WindowsUpdater.cs [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                return reinstall(filepath, update.LastVersion);
+            });
+        }
+
+        private static Boolean reinstall(String filepath, Version newVersion)
+        {
+            var code = Product.GetCode(filepath);
+            if (code == null) return false;
+
+            var succededUninstall = runOnMsi($"/x {code}", "uninstall");
```
Comment (wilkmaia) [inline PagarMe.Bifrost/Updates/WindowsUpdater.cs]: Before uninstalling, make sure the client is not transacting.

File: PagarMe.Bifrost.Certificates/PagarMe.Bifrost.Certificates.csproj (hunk 1 above)
Comment (darakeon) [inline PagarMe.Bifrost.Certificates/PagarMe.Bifrost.Certificates.csproj]: Done

File: PagarMe.Bifrost.Certificates/Generation/CertificateGenerator.cs (hunk 2 above)
Comment (darakeon) [inline PagarMe.Bifrost.Certificates/Generation/CertificateGenerator.cs]: Done

**Comment by darakeon** [comment]
Not working on Firefox

**Comment by darakeon** [comment]
The original code is not working either anymore, it's a issue with new Firefox.

**Pull request closed without merging**